src/python/pants/bin/pyoxidizer-howto.md

```diff
@@ -0,0 +1,17 @@
+NOTES:
+
+Run `./pants package --pyoxidizer-interpreter-constraints="['CPython==3.9.*']" src/python/pants/bin:pants_oxidized_experimental`
+
+The binary will be `dist/src.python.pants.bin/pants_oxidized_experimental/aarch64-apple-darwin/debug/install/pants_oxidized_experimental` -- this will not work on the pants repo itself (yet?)
```

Comment:
Because it doesn't embed some of the backends?

You can sometimes get around that by disabling the backends, and skipping config verification:
```console
--backend-packages='-["internal_plugins.test_lockfile_fixtures", "pants.backend.explorer"]' --no-verify-config
```

Reply:
There were things that didn't load early on in the process (that we may have since fixed); these notes haven't been hugely updated since I got things working initially. I'd still rather have a scratch file living in the repo rather than putting it in official docs (for now, anyway)